### ISSUE TITLE
CPU usage is through the roof

### ISSUE THREAD

**Issue by gajus** [kmagiera/babel-watch#46]
Something in `babel-watch` is being very greedy.

<img width="1919" alt="screen shot 2017-02-03 at 14 53 16" src="https://cloud.githubusercontent.com/assets/973543/22595700/88162ba2-ea20-11e6-9629-9483fcc2623f.png">

Platform info:

```
Darwin 16.3.0 x64
Node.JS 7.4.0
V8 5.4.500.45
NODE_ENV=production
Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz × 8
```

**Comment by qm3ster** [comment]
I got this too. Now I am just using
```json
{
    "dev": "yon watch:babel & yon  watch:run",
    "watch:babel": "babel -w src -d lib",
    "watch:run": "nodemon --watch lib lib/index.js"
}
```
yon is from `yarn-or-node` package.

**Comment by STRML** [comment]
This was e02dcc7b57d14a32e8f8726a9f2f600ed941c2fc, fixed as of `7.1.0`.
